src/themes/OLH/templates/core/base.html

```diff
@@ -149,7 +149,7 @@
             {% elif 'svg' in request.press_cover %}
                 <a href="{% url 'website_index' %}">{% svg request.press_cover %}</a>
             {% else %}
-                <a href="{% url 'website_index' %}">{% svg "static/common/img/sample/janeway.svg" %}</a>
+                <a href="https://janeway.systems">{% svg "static/common/img/sample/janeway.svg" "partial" %}</a>
```

Comment:
Ah, this is why we need the `partial` argument. Maybe the simplest thing is for the `svg` tag to return the actual path required for rendering (like django's `static` template tag)

Reply:
Because our external CSS manipulate the SVGs we can't just use a <img> tag we need the SVG in the HTML.